.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 /composer.lock
+/.idea
```

Comment:
please remove this

Reply:
it's generally a good rule to keep IDE files and OS specific files (like .ds_store) out of any git repository. These are "environment specific" files that should be ignored using a global gitignore file: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore